azure-pipelines.yml

```diff
@@ -2,21 +2,20 @@
 # Azure Pipeline NMODL settings
 #=============================================================================
 variables:
-- name: ReleaseWheelBuild
-  value: false
-- name: NightlyWheelBuild
-  value: false
+  # defined in UI
+  #- ReleaseWheelBuild: False
+  #- NightlyWheelBuild: False
```

Comment:
I think we can remove this

Reply:
OK, I thought I leave it more for documenting, this way we know that they are defined in UI and can be used inside yaml